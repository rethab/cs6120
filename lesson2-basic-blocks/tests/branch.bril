@main {
  v: int = const 1;
  c: bool = const false;
  br c .there .here;
.here:
  v: int = const 10;
.there:
  print v;
}
