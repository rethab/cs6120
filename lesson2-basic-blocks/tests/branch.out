digraph main {
  main;
  here;
  there;
  main -> here;
  here -> there;
}
